### ISSUE TITLE
 It's not supported in latest version of Defold (1.9.3)

### ISSUE THREAD

**Issue by bdshahab** [britzl/defold-clipboard#9]
I have 2 dependencies for my software that are not working in the new version of Defold!

one of them is clipboard and other is defos (which I reported before)

/clipboard/ext.manifest
	An error occurred while building Native Extensions, but the server provided no information.

**Comment by britzl** [comment]
I'm able to build this extension with Defold 1.9.3.

Which version of Defold are you using? Have you for some reason changed the URL to the build server? The Build Server url in editor preferences should be either https://build.defold.com or left blank.

https://defold.com/manuals/editor-preferences/#extensions

What does the editor log say?

https://defold.com/manuals/editor/#editor-logs

And the build server log?

https://defold.com/manuals/extensions/#build-server-logs

**Comment by bdshahab** [comment]
Thank you for this complete explanation.
The Build Server URL in editor preferences has not been changed.

I attached the editor log file by this message.

For Build server logs, I didn't use any native extensions.

My program is this:
https://github.com/bdshahab/iap_defold

[editor2.2024-09-26.log](https://github.com/user-attachments/files/17143442/editor2.2024-09-26.log)

**Comment by britzl** [comment]
> **I didn't use any native extensions**.

Yes you do. Both defold-clipboard and DefOS are native extensions and they need to be compiled and linked with the engine.

Are you able to reach https://build.defold.com/ from a browser? If not, then perhaps the URL is blocked in your firewall or something?

**Comment by bdshahab** [comment]
Oh! So those libraries are native extensions!
I cannot go to that URL! It's banned!🤦‍♂️

Error: Forbidden
Your client does not have permission to get URL /actuator/health from this server.

**Comment by britzl** [comment]
> I cannot go to that URL! It's banned!🤦‍♂️
> 
> Error: Forbidden Your client does not have permission to get URL /actuator/health from this server.

That explains it. We recently switched from AWS to GCP which may be the reason why it is no longer working for?

https://forum.defold.com/t/big-build-server-improvements/78244

**Comment by bdshahab** [comment]
Unfortunately, Google is the enemy of Iran! They banned Google Cloud Platform (GCP) for Iranian!
I cannot go to their URL either! (https://cloud.google.com)
